### DISCUSSION TITLE
How to re-transform ΔP(r) to ΔI(q)?

### DISCUSSION THREAD

**Discussion by sofiakap** [biosaxs-com/atsas-community#1958] [General]
Hello, 
I have a set of time-resolved scattering data and I would like to do a structural evaluation. In particular, I have transformed the ΔI(q) to ΔP(r) using one of the available softwares and a range of Dmax (so I can have a range of potential ΔP(r)). But I am not aware of a software that can do the opposite, meaning to transform the ΔP(r) back to ΔI(q), in order to choose which ΔP(r) describes the system best. The assumption is that if the experimental and transformed difference scattering do not deviate much from each other then I could conclude with relative certainty on the dimensions of the protein. Dmitri Svergun mentioned at the P12 User Meeting that this issue was addressed sometime in the past and that there is a way to do it. Could you please provide further details? Thank you.

**Comment by franke-biosaxs** [comment]
Could you please clarify what the Δ signifies in ΔI(q) and ΔP(r)? To me, Δ generally indicates some sort of difference, but I can't see any in this context.

Also: which software do you use to calculate the p(r)? If gnom, the resulting .out file does include the reconstructed I(q) already.

**Comment by sofiakap** [comment]
Dear Daniel, thank you for your response. Problem is solved, now. Just for the record, it is a pump-probe data set, hence the Delta, and I have used GNOM.
